tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -243,32 +251,142 @@ Scenario: 9. Get the order summary that includes a list of additional services
         | catalogue-solutions | incomplete | 0     |
         | additional-services | complete   | 2     |
         | funding-source      | incomplete |       |
+    And the order Section Status is incomplete
+
+@5291
+Scenario: 10. Get the order section setus is complete when Catalogue Solution viewed with > 0 items , Associated Service viewed with 0 items and Funding source is complete

```

Comment:
Please check formatting of the scenario description. The comma is surrounded by spaces.